Avoid double-quoting the resource path

andrewrabert opened the pull request:
The Request instance passed to add_response is quoted upon creation. When add_response invokes add_endpoint, the URL is quoted a second time. Thus, a _private_ method was created solely to register the endpoint without quoting.

dave-shawley commented:
@nvllsvm - rebase this over master and things should be good.  I removed the goofy test and adjusted the tornado pin.

andrewrabert committed:
Avoid double-quoting the resource path
The Request instance passed to add_response is quoted upon creation.
When add_response invokes add_endpoint, the URL is quoted a second time.
Thus, a _private_ method was created solely to register the endpoint
without quoting.

andrewrabert commented:
Done.
